### ISSUE TITLE
Waymo training configuration.

### ISSUE THREAD

**Issue by Hyeonjeong37** [Traffic-X/MonoLSS#18]
Hello! Thank you for your hard work!

Could you let me know the training configuration for the Waymo dataset such as learning rate, training epochs, and learning rate decay?

Thank you!
